events/2026-03-09-hts-workshop-freiburg.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: event
+title: "Workshop on high-throughput sequencing data analysis with Galaxy"
+
+# external: "https://galaxyproject.org/events/2025-03-10-galaxy-workshop-freiburg/"
+
+description: |
+    This course introduces scientists to the data analysis platform Galaxy. The course is designed for beginners; no prior programming skills are required.
+
+date_start: 2026-03-09
+date_end: 2026-03-13
+
+cost: free
+audience: Scientists with no or little Galaxy experience who want to analyse sequencing data.
+contact_email: schneidd@informatik.uni-freiburg.de
+# async: false
+mode: onsite
+
+registration:
+  link: https://forms.gle/1C2iHiEjmPJMiCaK6
+  deadline: 2026-02-08
+
+contributions:
+  organisers: [Sch-Da]
+  instructors: [dianichj, bgruening, Nilchia, wm75, paulzierep]
```

Comment:
regarding your helper/instructor questions, I would put everybody who helped out here, you can always put them in rough order of involvement level if you want. If you feel strongly we could always add another category for helpers, but basically this is just meant to give credit to people involved.